CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changes by Version
 ==================
 
+0.9.0 (2017-10-25)
```

Comment:
hold on, this looks like the full product version bump, but only includes UI changes. Don't we have backend changes going in as well?  

Either way, I'd like to separate UI and backend changes into subsections

Reply:
Yeah, it's only UI since that's the only relevant thing in the PR. I'll do a subsequent PR with the "Preparing to release etc." with the jaeger specific stuff

Reply:
ok. Normally you shouldn't be putting the actual date.